### PR TITLE
[Finishes #187355126] The products should stop listing if the expiry

### PR DESCRIPTION
**Pull request by Christian-Ishimwe**
### What does This PR Do
Removing products from listing when Expired

### Description of Task to be completed
if the product expire,The seller recieves the email, with the details of the product which have been expired, name, quantity , date of expiration and the Image of the product.
The product, will be flagged as expired and will no longer be listed on available products of our store.

### How should this be manually tested? 
First, register and verify  as seller, then create the collection, add a product, make sure to set also the expiration date, if the date of expiration reaches, the seller will recieve the email of the expired products.

Screenshot 2024-05-29 at 23 37 45

### CODE REVIEW

File: src/cronjobs/productsCron.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+});
+
+const sendExpiredEmail = (receiver: any, product: any) => {
+  let date = millisecondsToDate(product.expirationDate);
+  let mailOptions = {
+    from: '"Sender Name" <christianinja3@gmail.com>', // sender address
+    to: `${receiver.email}`, // list of receivers
+    subject: `Product Expire`, // Subject line
+    text: 'Hello world?', // plain text body
+    html: `
```
Comment (Celestin4) [inline src/cronjobs/productsCron.ts]: Can you move this html template in src/helps/MailTemplate.ts as it is on develop
image

File: src/database/migrations/20240415075217-create-user.ts
Comment (Celestin4) [inline src/database/migrations/20240415075217-create-user.ts:1]: wh for these changes

image

File: src/cronjobs/productsCron.ts
Comment (Celestin4) [inline src/cronjobs/productsCron.ts:1]: It will be better when you adapt to nodemail that is currently defined in project

**Review by Celestin4 — CHANGES_REQUESTED** [review]
That's is my comment

**Review by lilika67 — APPROVED** [review]

**Review by Celestin4 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [![HoundCI](https://img.shields.io/badge/reviewed%20by-Hound-%23a873d1.svg)](https://houndci.com)
 [![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn?branch=develop)
 
-[![CircleCI](https://circleci.com/gh/atlp-rwanda/e-commerce-ones-and-zeroes-bn/tree/develop.svg?style=svg)](https://app.circleci.com/pipelines/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/?branch=develop)
+[![CircleCI](https://circleci.com/gh/atlp-rwanda/e-commerce-ones-and-zeroes-bn/tree/fc-product-expire.svg?style=svg)](https://app.circleci.com/pipelines/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/?branch=fc-product-expire)
```
Comment (UmuhireJessie) [inline README.md]: These should be set to the base branch, i.e., `develop` branch

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Good work @Christian-Ishimwe, check my comment

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**